zap/src/main/java/org/parosproxy/paros/core/scanner/VariantJSONQuery.java

```diff
@@ -74,6 +74,9 @@ public boolean isValidContentType(String contentType) {
     @Override
     public void parseContent(String content) {
         sr = new SimpleStringReader(content);
+        if (content == null || content.length() == 0 || content.strip().length() == 0) {
```

Comment:
isEmpty()

Reply:
And is the strip really needed?

Reply:
`content` can't be null here.

Reply:
will fix it

Reply:
replaced the whole check with `isBlank()` and did put the `sr` instantiating after the check.

Reply:
That method is also Java 11.

Reply:
To stay Java 8 compatible you'll need to use apache lang or lang3 `StringUtils.isBlank(content)`

Reply:
Better the latter.

Reply:
So going to leave it with only the `isEmpty` check.